### ISSUE TITLE
[Feature Request] Reminders functionality for every note

### ISSUE THREAD

**Issue by Preformer** [clipto-pro/Android#52]
Should have functionality to pick date and time, basic repetition (daily, weekly, monthly, yearly) and advanced repetition with custom controls like - ![Custom](https://i.imgur.com/ePJXetw.jpg)
![Hourly](https://i.imgur.com/GkjmOjc.jpg)
![Weekly](https://i.imgur.com/W9LQmv8.jpg)
![Monthly](https://i.imgur.com/qrQHdVH.jpg)

Repetitions can be customized a bit more but this much is fine for a note app.

Reminders can be made either like - Ring once, Ring five times, Keep Ringing like during a call.

**Comment by atrashler** [comment]
Reminders will be supported in Clipto. Could you send a link on the app where you have taken these screenshots (if it is an app:)?

**Comment by Preformer** [comment]
@atrashler Yes! These are from [Tasks by Alex Baker](https://play.google.com/store/apps/details?id=org.tasks) it's a fantastic to do app for a personal single person use, probably the best; but no collaboration features. The screenshots are from the custom repetitions part of adding a reminder.

**Comment by d0x360** [comment]
Why would a clipboard manager need a reminder system?  Android has one already built in... Just set a reminder through the system itself.

**Comment by jstetten** [comment]
@atrashler, has there been any progress on built-in reminders at all? This is the one giant glaring omission that prevents this from being the one true all-in-one management app it nearly is. Reminders are a feature built into even the least-used, lowest-rated note taking apps out there (really, take a look).

As one lone user I know I can't convince you to add this, but I do know alarms and time-picker UIs are baked-in features of the Android libraries. Just the most very basic (ideally recurring) notification attribute would make your app much more productive, and as a brief look around the app store shows, is something users expect. More advanced bells and whistles can come later.

What do you think?

**Comment by d0x360** [comment]
Doesn't sound like a bad idea for those who want it. The question is... Do
enough people want a feature enough to code it and then test for bugs.
It's bugs that is usually the biggest concern because sometimes code
interacts in ways you didn't even think possible.

Guess we will see.

**Comment by jstetten** [comment]
Hey, @d0x360. As a developer (mostly for other platforms), I totally get it. I'd be more than happy to be an official or unofficial beta tester. I don't root my phones anymore, but I'd be glad to install test apks, adb in, send logcats, record screen caps, automate touch events, etc.

Not saying 24/7 all day every day, but y'know...I'll do what I can. :)

**Comment by d0x360** [comment]
I had a feeling since you're on here but it shouldn't effect things too
much (should it get implemented) I would think a calendar, clock and
triggers for them should be easy to do.  Although personally id prefer it
hooking into my Google calendar.
